kornia/geometry/transform/crop/crop2d.py

```diff
@@ -19,10 +19,10 @@
 
 def crop_and_resize(tensor: torch.Tensor, boxes: torch.Tensor, size: Tuple[int, int],
                     interpolation: str = 'bilinear', align_corners: bool = False) -> torch.Tensor:
-    r"""Extracts crops from the input tensor and resizes them.
+    r"""Extract crops from 2D images (4D tensor) and resize them.
 
     Args:
-        tensor (torch.Tensor): the reference tensor of shape BxCxHxW.
+        tensor (torch.Tensor): the 2D image tensor with shape (C, H, W) or (B, C, H, W).
```

Comment:
is it true `(C, H, W)` ? I would assume that for core functions  we don't branch that much and we receive BCHW. They user later can set B==1

Reply:
It is. This function converted CHW to BCHW internally, then it will revert it back to CHW. It can be removed if needed.

Reply:
I think it's better we make all the api homogeneous for BCHW